### DISCUSSION TITLE
Making player controls mushroomable

### DISCUSSION THREAD

**Discussion by pieterminack** [punxaphil/custom-sonos-card#178] [Q&A]
At first a lot of thanks for this fork of a Sonos card. This fits nearly all my needs. 

I wonder if it is possible to edit the player controls in any (simple) way to make it a little bit more integrated to the mushroom themes? For example like the mockup attached.
![sonos-player-mushroom-mockup](https://github.com/johanfrick/custom-sonos-card/assets/44088717/e37b04c7-ae8d-446b-9251-f954bc016928)

Thank you for any ideas.

**Comment by punxaphil** [comment]
Honestly looks a bit strange to me since the section buttons are not mushroom:y 🤔
